yoyodyne/models/pointer_generator.py

```diff
@@ -562,7 +561,7 @@ def forward(
             batch.source, self.source_embeddings, self.encoder
         )
         features_encoded, (h_features, c_features) = self.encode(
-            batch.features, self.feature_embeddings, self.feature_encoder
+            batch.features, self.features_embeddings, self.feature_encoder
```

Comment:
Looks like it is initialized as `self.feature_embeddings`, so one of them needs to be updated if I am not mistaken.

Reply:
This inconsistency bothers me so I'll put a consistency fix to this module in a separate PR so there's less irrelevant change in this PR.

Out of curiosity and while you're here, is there any good argument for having a separate features embedding in the pointer generator? It seems like one ought to be able to share one for source and features since they are already guaranteed to be disjoint.

Reply:
I believe this was added to support some changes for tracking features and tokens separately? This just avoids clashing indices, but I also think @bonham79 may have updated s.t. features/tokens can be tracked separately, but then combined in the collator?

Reply:
The indices can't crash anyways because we obfuscate features. But I guess it's harmless and maybe also fixed by Travis's PR so no worries.

Reply:
I think it was added here: https://github.com/CUNY-CL/yoyodyne/commit/f6cff4639916a2475fc330309c73ad8c918b3273 -- so guessing it was needed in some previous feature handling, but is no longer needed?